src/IssueViz.UnitTests/IssueVisualizationControl/IssueVisualizationViewModelTests.cs

```diff
@@ -81,8 +82,9 @@ private IssueVisualizationViewModel CreateTestSubject()
         [TestMethod]
         public void Ctor_InitializeWithExistingSelection()
         {
-            var newIssue = Mock.Of<IAnalysisIssueVisualization>();
-            var newFlow = CreateFlow(out var newLocation);
+            var newLocation = CreateLocation();
```

Comment:
I broke quite a lot of tests due to the change of `CurrentIssue`'s setter, that now tries to register to locations' event. All the places that I used `Mock.Of<IAnalysisIssueVisualization>()` had to be replaced with an object that has non-null `flows`.
So I just created normal methods CreateLocation, CreateFlow and CreateIssue, and used them in the tests.